1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/features/command/CommandManager.kt

```diff
@@ -1,9 +1,13 @@
 package net.ccbluex.liquidbounce.features.command
 
 import net.ccbluex.liquidbounce.features.command.commands.*
+import net.ccbluex.liquidbounce.features.module.Module
+import net.ccbluex.liquidbounce.features.module.ModuleInfo
+import net.ccbluex.liquidbounce.features.module.ModuleManager
 import net.ccbluex.liquidbounce.utils.ClientUtils
 import net.minecraftforge.fml.relauncher.Side
 import net.minecraftforge.fml.relauncher.SideOnly
+import org.reflections.Reflections
```

Comment:
Unnötig

Reply:
a new dependency isn't really need, but it makes it easier.